go/chat/helper.go

```diff
@@ -18,168 +18,71 @@ import (
 	"github.com/keybase/client/go/protocol/keybase1"
 )
 
-func SendTextByName(ctx context.Context, g *globals.Context, name string, topicName string,
-	membersType chat1.ConversationMembersType, ident keybase1.TLFIdentifyBehavior, text string,
-	ri chat1.RemoteInterface) error {
-	helper := newSendHelper(g, name, topicName, membersType, ident, ri)
-	return helper.SendText(ctx, text)
-}
-
-func SendMsgByName(ctx context.Context, g *globals.Context, name string, topicName string,
-	membersType chat1.ConversationMembersType, ident keybase1.TLFIdentifyBehavior, body chat1.MessageBody,
-	msgType chat1.MessageType, ri chat1.RemoteInterface) error {
-	helper := newSendHelper(g, name, topicName, membersType, ident, ri)
-	return helper.SendBody(ctx, body, msgType)
+func NewTeamNewConversationLocalArg(tlfName string, topicName *string, identifyBehavior keybase1.TLFIdentifyBehavior) chat1.NewConversationLocalArg {
+	return chat1.NewConversationLocalArg{
+		TlfName:          tlfName,
+		TopicName:        topicName,
+		IdentifyBehavior: identifyBehavior,
+		TopicType:        chat1.TopicType_CHAT,
+		TlfVisibility:    chat1.TLFVisibility_PRIVATE,
+		MembersType:      chat1.ConversationMembersType_TEAM,
+	}
 }
 
-type sendHelper struct {
+type SendHelper struct {
```

Comment:
Old architecture ran NewConversationLocal everytime a chat message was sent.
Here, it's only run once on initialization, which is why constructors can return errors.